dhcpv6/dhcpv6message.go

```diff
@@ -47,7 +50,7 @@ func GenerateTransactionID() (*uint32, error) {
 			return nil, err
 		}
 		if tid == nil {
-			return nil, fmt.Errorf("Error: got a nil Transaction ID")
+			return nil, fmt.Errorf("Got a nil Transaction ID")
```

Comment:
since we are here, make it lowercase as per https://github.com/golang/go/wiki/CodeReviewComments#error-strings